test/integration/apiserver.go

```diff
@@ -789,28 +789,25 @@ func offeringService(ctx context.Context, conn *grpc.ClientConn, account *catalo
 			LabelSelector: "test-label==offering1",
 		})
 		require.NoError(t, err)
+		nextEventType(t, watchClient, watch.Added)
+
 		// Update an offering object to get Modified event.
 		offering1.Spec.Metadata.ShortDescription = "test offering update"
 		require.NoError(t, managementClient.Update(ctx, offering1))
+		nextEventType(t, watchClient, watch.Modified)
```

Comment:
I am not sure if this will help, because sometimes the `Modified` event will be lost, @aborilov and I have seen this quite a lot

Reply:
They could be lost due to delay between create and delete?  

Otherwise we have to drop this test...